### PR TITLE
shutdownDuration of server assignments error

### PR DESCRIPTION
**Pull request by devincd**
Signed-off-by: devincd <505259926@qq.com>

fix typos and shutdownDuration of server assignments error.
If args.ShutdownDuration is equal to 20, s.shutdownDuration should also be 20, but s.shutdownDuration is 0.

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

Pull Request Attributes

Please check any characteristics that apply to this pull request. 

[ ] Does not have any changes that may affect Istio users.

### CODE REVIEW

File: pilot/pkg/bootstrap/options.go [hunk 1]
```diff
@@ -153,6 +153,7 @@ func (p *PilotArgs) applyDefaults() {
 	p.KeepaliveOptions = keepalive.DefaultOption()
 	p.RegistryOptions.DistributionTrackingEnabled = features.EnableDistributionTracking
 	p.RegistryOptions.DistributionCacheRetention = features.DistributionHistoryRetention
+	p.ShutdownDuration = 10 * time.Second
```
Comment (hzxuzhonghu) [inline pilot/pkg/bootstrap/options.go:156]: nit: it seems we never have a flag setting ShutdownDuration

**Review by hzxuzhonghu — APPROVED** [review]
lgtm

File: pilot/pkg/bootstrap/options.go (hunk 1 above)
Comment (devincd) [reply pilot/pkg/bootstrap/options.go:156]: yes, the ShutdownDuration parameter is then added to the flag, and some scenarios do need to configure this parameter.

Comment (devincd) [reply pilot/pkg/bootstrap/options.go]: I have already added a flag setting ShutdownDuration in pilot

File: releasenotes/notes/32201.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+apiVersion: release-notes/v2
+kind: feature
+area: networking
+issue:
+releaseNotes:
+- |
+  **Added** a flag setting ShutdownDuration in pilot and its default value is 10s.
```
Comment (shamsher31) [inline releasenotes/notes/32201.yaml:7]:
```suggestion
  **Added** a flag setting `ShutdownDuration` in pilot and its default value is 10s.
```

**Review by shamsher31 — APPROVED** [review]

**Pull request merged**